### PR TITLE
Remove fallback explorer

### PR DESCRIPTION
**Pull request by jinchung**
Fixes APP-315

### CODE REVIEW

File: src/App.js [hunk 1]
```diff
@@ -230,9 +229,7 @@ class OldApp extends Component {
             store.dispatch(explorerInitL2(network));
           }
         } else {
-          store.dispatch(
-            fetchOnchainBalances({ keepPolling: false, withPrices: false })
-          );
+          // TODO(jin) replace with new balance checker
```
Comment (jinchung) [inline src/App.js:232]: Leaving a comment for flagging TODO

File: src/hooks/useRefreshAccountData.ts [hunk 2]
```diff
@@ -33,9 +32,7 @@ export default function useRefreshAccountData() {
         ? dispatch(fetchWalletENSAvatars())
         : null;
       const getUniqueTokens = dispatch(uniqueTokensRefreshState());
-      const balances = dispatch(
-        fetchOnchainBalances({ keepPolling: false, withPrices: false })
-      );
+      // TODO(jin) - need to substitute for balances info
```
Comment (jinchung) [inline src/hooks/useRefreshAccountData.ts:35]: Leaving a comment for flagging TODO

File: src/redux/explorer.ts [hunk 3]
```diff
@@ -637,20 +542,15 @@ export const explorerInit = () => async (
...
   const providerUrl = provider?.connection?.url;
   checkForTheMerge(provider, network);
-  if (
-    isHardHat(providerUrl) ||
-    network !== Network.mainnet ||
-    forceFallbackProvider
-  ) {
-    return dispatch(fallbackExplorerInit());
+  if (isHardHat(providerUrl) || network !== Network.mainnet) {
+    return; // TODO JIN - used to do fallback init
```
Comment (jinchung) [inline src/redux/explorer.ts:553]: Leaving a comment for flagging TODO

File: src/redux/explorer.ts [hunk 4]
```diff
@@ -637,20 +542,13 @@ export const explorerInit = () => async (
...
   const providerUrl = provider?.connection?.url;
   checkForTheMerge(provider, network);
-  if (
-    isHardHat(providerUrl) ||
-    network !== Network.mainnet ||
-    forceFallbackProvider
-  ) {
-    return dispatch(fallbackExplorerInit());
+  if (isHardHat(providerUrl) || network !== Network.mainnet) {
+    return;
```
Comment (estrattonbailey) [inline src/redux/explorer.ts:551]: Should we alert ourselves if this happens or is this an impossible state?

**Review by estrattonbailey — APPROVED** [review]
LGTM, e2e might be flaking

File: src/redux/explorer.ts [hunk 5]
```diff
@@ -637,20 +542,13 @@ export const explorerInit = () => async (
...
   if (addressSocket || assetsSocket) {
     dispatch(explorerUnsubscribe());
-    dispatch(disableFallbackIfNeeded());
   }
 
-  // Fallback to the testnet data provider
-  // if we're not on mainnnet
   const provider = await getProviderForNetwork(network);
   const providerUrl = provider?.connection?.url;
   checkForTheMerge(provider, network);
```
Comment (skylarbarrera) [inline src/redux/explorer.ts:549]: unrelated but should we delete this?

**Review by skylarbarrera — APPROVED** [review]
burn it 🔥

File: src/redux/explorer.ts (hunk 5 above)
Comment (jinchung) [reply src/redux/explorer.ts:549]: yes I have it in my notes to follow up on this

File: src/redux/explorer.ts (hunk 4 above)
Comment (jinchung) [reply src/redux/explorer.ts:551]: I have a follow up note for testnet support (now we only support goerli and it isn't a big priority). I'm checking in on the hardhat branch for tests now.

**Review by jkadamczyk — APPROVED** [review]
Looks good to me!

**Comment by jinchung** [comment]
Closing this as it is now to be merged with the optimism explorer changes

**Pull request closed without merging**